packages/fetch-and-compile/test/fetch.test.ts

```diff
@@ -0,0 +1,81 @@
+import assert from "assert";
+import { describe, it } from "mocha";
+import Config from "@truffle/config";
+import { fetchAndCompile } from "../lib/index";
+import axios from "axios";
+import sinon from "sinon";
+const fixture: any = require("./fixture.js");
+
+function axiosMock(url: string, address: string, data: object) {
```

Comment:
How would you feel about changing this method name? You are not mocking axios, you are stubbing the axios get method. Maybe `stubAxiosGetMethod` or something like that.

Reply:
Yea great idea thanks.